logistics_planning_invoicing/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     ''',
     'author': 'Solvos',
     'license': 'LGPL-3',
-    'version': '13.0.2.0.1',
+    'version': '13.0.2.1.1',
```

Comment:
And fix this bump version:
```suggestion
    'version': '13.0.2.1.0',
```